protos/bottle/catalog/v1/addon.proto

```diff
@@ -6,6 +6,8 @@ message Addon {
   enum AddonCategory {
     ADDON_CATEGORY_UNSPECIFIED = 0;
     ADDON_CATEGORY_RUSH_ASSEMBLY = 1;
+    ADDON_CATEGORY_HANDLING = 2;
+    ADDON_CATEGORY_EWASTE = 3;
```

Comment:
This is the ewaste fee surcharge we apply to laptops and monitors. I figure we can split it off like what we do with rush assembly.